tests/sandbox/test_sandbox.py

```diff
@@ -204,6 +139,8 @@ def test_update_replace_given_name(self, additional_headers):
         helpers.check_response_headers(update_response, additional_headers)
 
     def test_update_suffix_from_name(self, additional_headers):
+        # Prefer header depreated, check that header is ignored
```

Comment:
Minor grammar change deprecated* :). Also, do we need to make any changes to sandbox?

Reply:
woops, thanks for pointing that out! 
I've added a policy to sanbox proxy to remove the the "prefer" header if its passed in by the client and did a minor tweak to the tests to check that its being ignored. Other than that nothing else needs to be changed on sandbox :-)